core/tests/unittest.py

```diff
@@ -188,16 +200,12 @@ def run_class(c, test_result):
     print('class', c.__qualname__)
     for name in dir(o):
         if name.startswith("test"):
-            is_async = name.startswith("test_async")
```

Comment:
why this change?

i was not aware of this `test_async_` functionality, if it works, maybe we should use it instead of my hacks?